packages/redux/src/recommendations/__tests__/reducer.test.ts

```diff
@@ -1,11 +1,13 @@
-import { mockRecommendationsStrategy } from '../../../tests/__fixtures__/initialReduxState';
+import { INITIAL_STATE } from '../reducer';
+import { mockRecommendationsStrategy } from 'tests/__fixtures__/recommendations';
 import reducer, { actionTypes } from '..';
+import type { State } from '../types';
 
 let initialState;
```

Comment:
This has `any` type, so typescript cannot know if the properties you are accessing from initialState are there or not. Can you add a type to initialState?

Reply:
Source updated, can you check again.